### PR TITLE
Tool fixes and the sort

### PR DESCRIPTION
**Pull request by funkychicken493**

### CODE REVIEW

File: denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc [hunk 1]
```diff
@@ -6,18 +6,18 @@ Amethyst_hoe:
...
-    - Far more brittle than diamond.
+    - <&r>Far more brittle than diamond.
   display name: <&f>Amethyst Hoe
   recipes:
     1:
       type: shaped
       output_quantity: 1
       input:
-      - Amethyst_ingot|Amethyst_ingot|air
+      - air|amethyst_ingot|amethyst_ingot
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc]: Does not match vanilla recipe

File: denizen_scripts/HeroCraft/items/tools/copper_tools/copper_axe.dsc [hunk 2]
```diff
@@ -20,4 +20,11 @@ copper_axe:
       - copper_ingot|copper_ingot|air
       - copper_ingot|stick|air
       - air|stick|air
+    2:
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/items/tools/copper_tools/copper_axe.dsc:23]: Does not match Vanilla

File: denizen_scripts/HeroCraft/items/tools/copper_tools/copper_hoe.dsc [hunk 3]
```diff
@@ -20,4 +20,11 @@ copper_hoe:
       - copper_ingot|copper_ingot|air
       - air|stick|air
       - air|stick|air
+    2:
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/items/tools/copper_tools/copper_hoe.dsc:23]: Does not match vanilla

File: denizen_scripts/HeroCraft/items/tools/steel_tools/steel_axe.dsc [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
     1:
       type: shaped
       output_quantity: 1
       input:
-      - Steel_ingot|Steel_ingot|air
-      - Steel_ingot|stick|air
+      - steel_ingot|steel_ingot|air
+      - steel_ingot|stick|air
+      - air|stick|air
+    2:
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/items/tools/steel_tools/steel_axe.dsc:23]: Does not match Vanilla

File: denizen_scripts/HeroCraft/items/tools/steel_tools/steel_hoe.dsc [hunk 5]
```diff
@@ -1,23 +1,30 @@
...
   recipes:
     1:
       type: shaped
       output_quantity: 1
       input:
-      - Steel_ingot|Steel_ingot|air
+      - steel_ingot|steel_ingot|air
+      - air|stick|air
+      - air|stick|air
+    2:
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/items/tools/steel_tools/steel_hoe.dsc:23]: Does not match Vanilla

**Comment by Ph4i1ur3** [comment]
Ignore the Vanilla recipe comments, this was resolved in Discord

File: denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc (hunk 1 above)
Comment (Ph4i1ur3) [reply denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc]: This should actually be a second recipe, not overwriting the first

Comment (funkychicken493) [reply denizen_scripts/HeroCraft/items/tools/amethyst_tools/amethyst_hoe.dsc]: Will fix

**Pull request merged**